servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -4417,7 +4417,7 @@ void RendererSceneRenderRD::_update_volumetric_fog(RID p_render_buffers, RID p_e
 
 		uint32_t cluster_screen_width = (rb->width - 1) / cluster_size + 1;
 		uint32_t cluster_screen_height = (rb->height - 1) / cluster_size + 1;
-		params.cluster_type_size = cluster_screen_width * cluster_screen_height * (32 + 32);
+		params.cluster_type_size = cluster_screen_width * cluster_screen_height * (16 + 32);
```

Comment:
I wonder if this should be the same as when doing the regular scene render i.e.: https://github.com/godotengine/godot/blob/547c27077721cf024031bedd69e55268c8d6b410/servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp#L626

Reply:
You're right, this should be (params.max_cluster_element_count_div_32 + 32). Changed and tested it accordingly, seems to work as expected.